strawberry/dataloader.py

```diff
@@ -78,6 +88,9 @@ def load(self, key: K) -> Awaitable[T]:
 
         return future
 
+    def load_many(self, keys: Iterable[K]) -> Awaitable[List[T]]:
+        return gather(*[self.load(key) for key in keys])
+
```

Comment:
Minor, nitpicky suggestion:
```suggestion
    def load_many(self, keys: Iterable[K]) -> Awaitable[List[T]]:
        return gather(*map(self.load, keys))
```

Reply:
Updated